src/pages/commits/CommitsWrapper.tsx

```diff
@@ -57,6 +58,7 @@ export const CommitsWrapper: React.FC<Props> = ({
                   author={version.author}
                   message={version.message}
                 />
+                <BuildVariantAccordion buildVariants={version.buildVariants} />
               </ColumnContainer>
```

Comment:
I can't select the entire block of code but this might better suited as its own component. To help group all of the components that represents an active version on the waterfall.

Reply:
Good idea. I will create `ActiveCommit`. Since this is no longer "accorion" I will also rename couple of files. I also think it makes more sense now if we have `buildVariantCard` as its own component instead of having `buildVariantCardContainer`